imports/plugins/included/product-variant/client/templates/products/productGrid/controls.js

```diff
@@ -21,6 +21,13 @@ Template.gridControls.onRendered(function () {
 });
 
 Template.gridControls.helpers({
+  getTour3() {
+    const steps = {
```

Comment:
es lint configuration says that "return statement should not contain assignment"